backend/src/main/conf/backend-conf.json

```diff
@@ -80,7 +80,7 @@
       "idletime.secs": 60,
       "queue.maxsize": 50
     },
-    "base.dir": "profiles"
+    "aggregatedProfiles.baseDir": "profiles"
```

Comment:
This one is inside the storage config block here, which is different from that in case of userapi now.